policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Maine personal exemption amount
+metadata:
+  unit: currency-USD
+  label: Maine Personal Exemption Amount
+  reference:
+    - title: Instructions for Form 1040ME
```

Comment:
Could you add the legislative reference for this one too? I'll merge after that.

Reply:
Actually, it looks like the amount has gone up to $4,450 in 2022 and is scheduled to go up to $4,700 in 2023. Does it seem like I'm understanding that correctly? If so, should I go ahead and put in the different period/amounts?
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ta_nov2022_vol32_iss22.pdf
That page links to (2022 and 2023 resp):
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2022.pdf
https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2023.pdf

Reply:
Yes please do! And also the legal code for the pre-inflation-adjustment amount.

Reply:
I can also add the old phaseout starts, but as far as I can tell those only changed in 2021 and we didn't do that initially to not imply that we had historical coverage?
https://www.mainelegislature.org/legis/statutes/36/title36sec5126-A.html

Reply:
Yeah we haven't yet set a standard here. Filed https://github.com/PolicyEngine/policyengine-us/issues/1805 to do that.

I meant to cite the code without entering the historical value, but this works. Thanks - merging.